source/clear-linux/guides/maintenance/Telemetry-Enable-Disable.rst

```diff
@@ -0,0 +1,111 @@
+.. _telemetry-enable-disable:
+
+Enable and Disable Telemetry in Clear Linux
+###########################################
+
+|CLOSIA| includes a telemetry solution as part of the OS that records events
+of interest and reports them back to the development team via the telemetrics
+daemon, :command:`telemd`.  This functionality is maintained in the
+``telemetrics`` software bundle.
+
+.. Note::
+	The telemetrics functionality adheres to `Intel’s privacy policies`_
+	regarding the collection and use of Personally Identifiable Information
+	(PII) and is Open Source. Specifically, no intentionally identifiable
+	information about the user or system owner is collected.
+
+End users may enable or disable the telemetry component of |CL| or even
+redirect where records go if they wish to collect records for themselves.
+
+Install the telemetry software bundle
+*************************************
+
+During the initial installation of |CL| you are requested to join the
+Stability Enhancement Program and allow |CLOSIA| to collect anonymous reports
+to improve system stability.  If you chose not to join this program at that
+time then the telemetry software bundle was not added to your system.
+
+To install the telemetry bundle, enter the following command as either the
+root user or with :command:`sudo` priveleges:
+
+.. code-block:: console
+
+	sudo swupd bundle-add telemetrics
+
+This will add the telemetrics-client to your system and you will automatically
+be opted-in for the service.
+
+Enable Telemetry
+****************
+
+To start telemetry on your system run the following command:
+
+.. code-block:: console
+
+	sudo telemctl start
+
+This will enable and start the :command:`telemd` daemon and your system will
+begin to send telemetry data to the server defined in the file
+:file:`/etc/telemetrics/telemetrics.conf` and if this file does not exist it
+will use the file :file:`/usr/share/defaults/telemetrics/telemetrics.conf`.
+
+Disable Telemetry
+*****************
+
+To disable the telemetry daemon run the following command:
+
+.. code-block:: console
+
+	sudo telemctl stop
+
+Opt-out of telemetry
+********************
+
+To stop sending telemetrics data from your system, opt out of the
+telemetry service:
+
+.. code-block:: console
+
+	sudo telemctl opt-out
+
+This will create the file :file:`/etc/telemetrics/opt-out` and stop the
+telemetry services.
+
+Opt-in to telemetry
+*******************
+
+Conversely, to opt-in to the telemetry services, simply enter the opt-in
+command and start the service:
+
+.. code-block:: console
+
+	sudo telemctl opt-in
+	
+This will remove the file :file:`/etc/telemetrics/opt-out` file if it exists
+and start the telemetry services.
+
+.. Note::
+	To opt-in but not immediately start telemetry services, you will need to
+	run the command :command:`sudo telemctl stop` after the :command:`opt-in`
+	command is entered.  Once you are ready to start the service, enter the

```

Comment:
Double space.

Reply:
Are you saying not to use double space after the end of a sentence?